src/utils/numbers/toBigNumber.js

```diff
@@ -4,6 +4,7 @@
 import BigNumber from 'bignumber.js'
 
 function toBigNumber(value: BigNumber | string | number | void, base?: number = 10): BigNumber {
+  BigNumber.config({ EXPONENTIAL_AT: 1e+9 })
```

Comment:
are you sure that this should be executed each toBigNumber, there will be millions ones?
and why it is needed?

Reply:
It is need to fix issue with BigNumber.toString() -> 1e-6 now seems like 0.0000001
It is fastest solution
If we add it before the function it will breaks on HMR

Reply:
This function WILL not work propertly with gas price user input

Reply:
https://github.com/jibrelnetwork/jwallet-web/blob/d6986eaa0e49ee5dac09d7002abb7268a3e964ea/src/utils/numbers/formatBalance.js#L29

We shouldn't show values like '>0.01'
```
    if (valueBN.lt(0.01)) {
      return '>0.01'
    }
```

In the user input

Reply:
And it should be `<` and narrow no-break space `&#8239;` in original method.

Bug added: https://jibrelnetwork.atlassian.net/browse/JWWEB-445

Reply:
gasPrice will be never float value